docs/Customization/RoCC-Accelerators.rst

```diff
@@ -46,6 +50,47 @@ the ``busy`` signal, which indicates when the accelerator is still handling an i
 and the ``interrupt`` signal, which can be used to interrupt the CPU.
 
 Look at the examples in ``generators/rocket-chip/src/main/scala/tile/LazyRoCC.scala`` for detailed information on the different IOs.
+There is also more information about each of the signals in `the RoCC Documentation written by UCSD <https://docs.google.com/document/d/1CH2ep4YcL_ojsa3BVHEW-uwcKh1FlFTjH_kg5v8bxVw/edit>`_, although it is updated out of tree and may be out of date.
+
+
+Accessing Memory via L1 Cache
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+A RoCC accelerator can access memory through the L1 Cache of the core it is attached to.
+This is typically a good idea, as a RoCC instruction that passes in an address likely has accessed it recently.
```

Comment:
Can you phrase this as "This is a simpler interface for accelerator architects to implement, but will generally have lower achievable throughput than a dedicated TileLink port"